Sources/Citadel/TTY/Client/TTY.swift

```diff
@@ -222,20 +227,30 @@ extension SSHClient {
     /// Executes a command on the remote server. This will return the output stream of the command. If the command fails, the error will be thrown.
     /// - Parameters:
     /// - command: The command to execute.
-    public func executeCommandStream(_ command: String) async throws -> AsyncThrowingStream<ExecCommandOutput, Error> {
+    /// - inShell:  Whether to request the remote server to start a shell before executing the command.
+    public func executeCommandStream(_ command: String, inShell: Bool = false) async throws -> AsyncThrowingStream<ExecCommandOutput, Error> {
         var streamContinuation: AsyncThrowingStream<ExecCommandOutput, Error>.Continuation!
         let stream = AsyncThrowingStream<ExecCommandOutput, Error> { continuation in
             streamContinuation = continuation
         }
         
-        let handler = ExecCommandHandler(logger: logger) { output in
+        var hasReceivedChannelSuccess = false
```

Comment:
This is fine, but I think Swift will complain about Sendability here at some point...